app/uk/gov/hmrc/exports/services/mapping/declaration/ExitOfficeBuilder.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services.mapping.declaration
+
+import javax.inject.Inject
+import uk.gov.hmrc.exports.models.Choice.AllowedChoiceValues
+import uk.gov.hmrc.exports.models.declaration.ExportsDeclaration
+import uk.gov.hmrc.exports.models.declaration.officeOfExit.{OfficeOfExitStandard, OfficeOfExitSupplementary}
+import uk.gov.hmrc.exports.services.mapping.ModifyingBuilder
+import wco.datamodel.wco.dec_dms._2.Declaration
+import wco.datamodel.wco.dec_dms._2.Declaration.ExitOffice
+import wco.datamodel.wco.declaration_ds.dms._2._
+
+class ExitOfficeBuilder @Inject()() extends ModifyingBuilder[ExportsDeclaration, Declaration] {
+  override def buildThenAdd(model: ExportsDeclaration, declaration: Declaration): Unit = model.choice match {
+    case AllowedChoiceValues.StandardDec =>
+      model.locations.officeOfExit
+        .map(OfficeOfExitStandard(_))
+        .map(createExitOfficeFromStandardJourney)
+        .foreach(declaration.setExitOffice)
+
+    case AllowedChoiceValues.SupplementaryDec =>
+      model.locations.officeOfExit
+        .map(OfficeOfExitSupplementary(_))
+        .map(createExitOfficeFromSupplementaryJourney)
+        .foreach(declaration.setExitOffice)
+  }
+
+  private def createExitOfficeFromStandardJourney(data: OfficeOfExitStandard): Declaration.ExitOffice = {
```

Comment:
Looks to me like these two `create` methods are identical, we can just pass in an `OfficeOfExit` and remove the other

Reply:
Fixed this for you @PauloPortugal